source/brailleDisplayDrivers/albatross.py

```diff
@@ -0,0 +1,822 @@
+# A part of NonVisual Desktop Access (NVDA)

```

Comment:
The complexity of this display driver makes this difficult to review.

Is it possible to split this up into it's own submodule? Potential example layout:
```
brailleDisplayDrivers
- albatross
-- __init__.py
-- displayDriver.py
-- constants.py
-- threading.py